keywords/MobileRestClient.py

```diff
@@ -2266,3 +2266,53 @@ def get_revs_num_in_history(self, url, db, doc_id, auth=None):
         doc = self.get_doc(url, db, doc_id, auth)
         logging.debug(doc)
         return doc["_revisions"]["ids"]
+
+    def sgCollect_restCall(self, sg_host, redact_level=None, redact_salt=None, output_directory=None, upload=False, upload_host=None, customer=None, ticket=None):
+        """
+        set all parameters to the dictionary and do post
+        """
+        body = {}
+        body["upload"] = upload
+        if redact_level is not None:
+            body["redact_level"] = redact_level
+        if redact_salt is not None:
+            body["redact_salt"] = redact_salt
+        if output_directory is not None:
+            body["output_dir"] = output_directory
+        if upload_host is not None:
+            body["upload_host"] = upload_host
+        if customer is not None:
+            body["customer"] = customer
+        if ticket is not None:
+            body["ticket"] = ticket
+        resp = self._session.post("http://{}:4985/_sgcollect_info".format(sg_host), data=json.dumps(body))
+        log_r(resp)
+        return resp
+
+    def sgCollect_info(self, sg_host, redact_level=None, redact_salt=None, output_directory=None, upload=False, upload_host=None, customer=None, ticket=None):
+        """
+        Get sgCollect info using rest Api call by passing params
+        """
+        resp = self.sgCollect_restCall(sg_host, redact_level=redact_level, redact_salt=redact_salt, output_directory=output_directory, upload=upload, upload_host=upload_host, customer=customer, ticket=ticket)
+        print "response of sgcollect info is ", resp.content
```

Comment:
change it to log_info from print

Reply:
removed it as it is not required to print